torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -343,14 +346,14 @@ def _unshard(
     :meth:`summon_full_params` and are using mixed precision, then they are
     forced to full precision.
 
-    Postcondition: Each handle's ``FlatParameter`` 's data is the padded
+    Postcondition: handle's ``FlatParameter`` 's data is the padded
     unsharded flat parameter on the compute device.
     """
-    if not handles:
+    if not handle:
         return
     any_ran_pre_unshard = False
     with state._device_handle.stream(pre_unshard_stream):
-        for handle in handles:
+        if handle:
```

Comment:
nit: We do not need this `if handle:` since we return early on `if not handle:`.